GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -271,8 +321,22 @@ gdjs.WebsocketDebuggerClient.prototype.sendProfilerOutput = function(
   );
 };
 
-// This is an alternative to JSON.stringify that ensure that circular reference
-// are replaced by a placeholder.
+/**
+ * A Function used to replace Cycles with a default name.
```

Comment:
no capital to function and cycle. 
Nitpicking: Prefer "circular references", as this is the term used in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
"A function used to replace circular reference with a new value."